openedx/core/djangoapps/course_groups/models.py

```diff
@@ -63,7 +63,8 @@ class Meta:
     # For now, only have group type 'cohort', but adding a type field to support
     # things like 'question_discussion', 'friends', 'off-line-class', etc
     COHORT = 'cohort'  # If changing this string, update it in migration 0006.forwards() as well
-    GROUP_TYPE_CHOICES = ((COHORT, 'Cohort'),)
+    TEAM = 'team'
+    GROUP_TYPE_CHOICES = ((COHORT, 'Cohort'), (TEAM, 'Team'),)
```

Comment:
[question] if we already have a way of grouping users (teams) why do we need this mapping?